.pipelines/pipeline.user.windows.official.yml

```diff
@@ -0,0 +1,80 @@
+environment:
+  host:
+    os: 'windows'
+    flavor: 'server'
+    version: '2019'
+  runtime:
+    provider: 'appcontainer'
+    image: 'cdpxwin1809.azurecr.io/global/vse2019:latest'
+    source_mode: 'link'
+
+version:
+  name: 'ApplicationInsights-dotnet'
+  major: 0
+  minor: 0
+  system: 'custom'
+
+signing_options:
+  profile: 'external_distribution'
+
+restore:
+  commands:
+    - !!defaultcommand
+      name: 'Hello World'
+      command: '.pipelines\00_helloworld.cmd'
+      arguments: 'official'
+      fail_on_stderr: false
+    - !!defaultcommand
+      name: 'Show .NET versions'
+      command: '.pipelines\00_list_versions.cmd'
+      fail_on_stderr: false
+    - !!defaultcommand
+      name: 'Install dependencies'
+      command: '.pipelines\01_restore.cmd'
+      fail_on_stderr: false
+
+build:
+  commands:
+    - !!buildcommand
+      name: 'Dotnet Build'
+      command: '.pipelines\02_build.cmd'
+      arguments: '-p:PublicRelease=True -p:StableRelease=True -p:OfficialRelease=True'
+      fail_on_stderr: false
+      artifacts:
+        - from: '.\bin\Release\src'
+          include:
+            - '**/*.dll'
```

Comment:
it seems `\` and `/` are mixed up, consider using `/`.

Reply:
done